script/bootstrap

```diff
@@ -7,5 +7,6 @@ set -e
 cd "$(dirname "$0")/.."
 
 echo "Installing development dependencies..."
+python3 -m pip install --upgrade pip
```

Comment:
This cannot be added as such, as `pip` is pinned in our requirements as well. This will skip/bypass this pin.

Reply:
Okay, I see. I'll be closing this PR